### ISSUE TITLE
License of the app

### ISSUE THREAD

**Issue by bangarharshit** [adrianblp/Materialist#1]
The  app is amazing. I wanted to know the license of the app. It will be great if you can add one.

**Comment by aladagemre** [comment]
Yes, that would be nice. I want to fork it and customise it if it's free software.

**Comment by TurboKia** [comment]
Would be great if you could add a delete to completed items. Then it would be pretty much perfect.
